### PR TITLE
Robust locales aka Cannot start mongo CLI

### PR DESCRIPTION
**Pull request by ppp0**
Regression via https://github.com/cargomedia/puppet-packages/pull/1367

On wheezy machine www3:
```
$ mongo
Failed global initialization: BadValue Invalid or no user locale set. Please ensure LANG and/or LC_* environment variables are set correctly.
```

Where `LANG` is `en_US.UTF-8`.

While this works:
```
$ LANG=en_US mongo
MongoDB shell version: 2.6.12
```

### CODE REVIEW

**Comment by ppp0** [comment]
@kris-lab please review

File: modules/locale/manifests/init.pp [hunk 1]
```diff
@@ -1,9 +1,25 @@
 class locale($lang = 'en_US.UTF-8') {
 
-  file { '/etc/default/locale':
-    content => template("${module_name}/locale.erb"),
-    owner   => '0',
-    group   => '0',
-    mode    => '0644',
+  exec { "Uncomment ${lang} if needed":
```
Comment (kris-lab) [inline modules/locale/manifests/init.pp]: Is it needed? I remember that `update-locale` was commenting/uncommenting some line. Can you confirm?

Comment (ppp0) [inline modules/locale/manifests/init.pp]: Uhm.. the file affected by uncommenting is `/etc/locale.gen` - afaik the uncommenting has to be done either "manually" or via `dpkg-reconfigure locales`

Comment (ppp0) [inline modules/locale/manifests/init.pp]: I added changes. Uncommenting is now for Wheezy only. And it is necessary

**Comment by ppp0** [comment]
@kris-lab please review, finally it's green

**Comment by ppp0** [comment]
closes https://github.com/cargomedia/puppet-cargomedia/issues/1267

**Comment by kris-lab** [comment]
As discussed to be on the safe side let's change
```
if ($::lsbdistid != 'Ubuntu') {
```
with checking for `Dabian` as we are aware only of this case.

rest lgtm

**Pull request merged**